internal/buffer/runtime_go1.8.s

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build amd64 arm64 ppc64 ppc64le
+// +build 386 amd64 arm arm64 ppc64 ppc64le
```

Comment:
Are you sure that the frame sizes listed below (`$0-16`, `$0-24`) are correct on these platforms? It seems like they're double what they should be...